.release-it.json

```diff
@@ -0,0 +1,23 @@
+{
+    "hooks": {
+      "before:init": ["npm test"]
+    },
+    "git": {
+      "requireCleanWorkingDir": true,
+      "commitMessage": "chore: release ${version}",
```

Comment:
I don't think a commitizen style commit message is necessary here, otherwise the release notes/`CHANGELOG.md` might always have an extra point  about `release ${version}`. See the [example repo setup](https://github.com/Amsterdam/ee-docs/blob/develop/.release-it.json)